openmc/mgxs/library.py

```diff
@@ -363,6 +372,7 @@ def load_from_statepoint(self, statepoint):
 
         self._sp_filename = statepoint._f.filename
         self._openmc_geometry = statepoint.summary.openmc_geometry
+        self._keff = statepoint.k_combined[0]
```

Comment:
If multi-group cross sections are generated for a fixed source problem, `statepoint.k_combined[0]` would raise TypeError since k_combined is None.